src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -453,19 +452,22 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
         <InnerFormContainer>
           <FormField title="Description">
             <Tooltip text="Click to edit channel description">
-              <TextArea
+              <Controller
                 name="description"
-                placeholder="Description of your channel to share with your audience"
-                rows={8}
-                ref={(ref) => {
-                  if (ref) {
-                    register(ref, textFieldValidation({ name: 'Description', minLength: 3, maxLength: 1000 }))
-                    descriptionRef.current = ref
-                  }
-                }}
-                maxLength={1000}
-                error={!!errors.description}
-                helperText={errors.description?.message}
+                control={control}
+                rules={textFieldValidation({ name: 'Description', minLength: 3, maxLength: 1000 })}
+                render={({ field: { value, onChange } }) => (
+                  <TextArea
+                    placeholder="Description of your channel to share with your audience"
+                    rows={8}
+                    ref={descriptionRef}
+                    value={value}
+                    onChange={onChange}
+                    maxLength={1000}
+                    error={!!errors.description}
+                    helperText={errors.description?.message}
+                  />
+                )}
```

Comment:
Why did we move to a `Controller` here? Is that because of sharing `ref` usage? I'm pretty sure react-hook-form@7 has to have some kind of solution for that without using `Controller`?

Reply:
Yes, it is because of the `ref` usage. And yes, it has solution for sharing refs but it would require registering description above the return (and not in the component itself) with something like:
`const { ref, ...rest } = register('description');`

 Thought it might be confusing to register a component in a different place than others